samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/bean/evaluation/SubmissionNavBean.java

```diff
@@ -0,0 +1,96 @@
+package org.sakaiproject.tool.assessment.ui.bean.evaluation;
+
+import java.io.Serializable;
+import java.text.DateFormat;
+import java.util.ArrayList;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Locale;
+import java.util.Map;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+
+import javax.faces.bean.ManagedBean;
+import javax.faces.bean.SessionScoped;
+import javax.faces.model.SelectItem;
+
+import org.sakaiproject.time.api.UserTimeService;
+import org.sakaiproject.tool.assessment.data.dao.grading.AssessmentGradingData;
+import org.sakaiproject.tool.assessment.ui.listener.util.ContextUtil;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.web.context.support.SpringBeanAutowiringSupport;
+
+import lombok.Data;
+
+import lombok.extern.slf4j.Slf4j;
+
+/* For evaluation: Submission navigation backing bean. */
+@Slf4j
+@ManagedBean(name="submissionNav")
+@SessionScoped
+@Data
+public class SubmissionNavBean extends SpringBeanAutowiringSupport implements Serializable {
+
+
+    private static final long serialVersionUID = 5517587781720762297L;
+
+    private String nextGradingId;
+    private String currentGradingId;
+    private String previousGradingId;
+    private Boolean displaySubmissionDate;
+    private SelectItem[] submissionsSelection;
+
+    @Autowired
+    @Qualifier("org.sakaiproject.time.api.UserTimeService")
+    private UserTimeService userTimeService;
+
```

Comment:
extra space

Reply:
This was intentional. Two blank lines before constructors. https://www.baeldung.com/java-clean-code#4-whitespaces
Should I remove it?

Reply:
Nah, fine then